<a href="https://colab.research.google.com/github/Vania12San/analisis-datos-i/blob/main/portafolio_adi_vasc_p01.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## TEMA ELEGIDO

MERCADOTECNIA ESTRATÉGICA- "QUIEN ES QUIEN EN LOS PRECIOS"


In [ ]:
library(readr)

In [ ]:
datos <- read_csv("11-2025_01.csv")

Rows: 486182 Columns: 13
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (12): producto, presentacion, marca, categoria, catalogo, fecha_registro...
dbl  (1): precio

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


In [ ]:
datos

producto,presentacion,marca,categoria,catalogo,precio,fecha_registro,cadena_comercial,giro,nombre_comercial,direccion,estado,municipio
<chr>,<chr>,<chr>,<chr>,<chr>,<dbl>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>
Aceite,Botella 850 Ml. Vegetal,Precíssimo,Aceites y Grasas Veg. Comestibles,Pacic,26.50,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Arroz,Bolsa 900 Gr. Extra. Grano Largo,Precíssimo,Arroz y Cereales Preparados,Pacic,16.90,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Atún,"Lata 140 Gr. Desmenuzado en Aceite. Standard, Sin Soya",Fresh Label,Pescados y Mariscos en Conserva,Pacic,17.90,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Atún,"Lata 140 Gr. Desmenuzado en Agua. Standard, Sin Soya",Fresh Label,Pescados y Mariscos en Conserva,Pacic,17.90,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Azúcar,Bolsa Plástico 900 Gr. Estándar,Precíssimo,Azucar,Pacic,25.90,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Azúcar,Bolsa Plástico 900 Gr. Estándar,Zulka,Azucar,Pacic,24.90,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Carne Cerdo,1 Kg. Granel. Chuleta o Chuleta Natural o Fresca/chuleta de Puerco Entrecortada o Lomo Deshuesado de Puerco o Chuleta Natural Lomo de Puerco,S/m,Carne y Visceras de Cerdo,Pacic,128.00,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Carne Cerdo,1 Kg. Granel. Pulpa en Trocitos,S/m,Carne y Visceras de Cerdo,Pacic,88.00,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Carne Cerdo,1 Kg. Granel. Pulpa en Trozo,S/m,Carne y Visceras de Cerdo,Pacic,88.00,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes


In [ ]:
head(datos)

producto,presentacion,marca,categoria,catalogo,precio,fecha_registro,cadena_comercial,giro,nombre_comercial,direccion,estado,municipio
<chr>,<chr>,<chr>,<chr>,<chr>,<dbl>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>
Aceite,Botella 850 Ml. Vegetal,Precíssimo,Aceites y Grasas Veg. Comestibles,Pacic,26.5,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Arroz,Bolsa 900 Gr. Extra. Grano Largo,Precíssimo,Arroz y Cereales Preparados,Pacic,16.9,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Atún,"Lata 140 Gr. Desmenuzado en Aceite. Standard, Sin Soya",Fresh Label,Pescados y Mariscos en Conserva,Pacic,17.9,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Atún,"Lata 140 Gr. Desmenuzado en Agua. Standard, Sin Soya",Fresh Label,Pescados y Mariscos en Conserva,Pacic,17.9,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Azúcar,Bolsa Plástico 900 Gr. Estándar,Precíssimo,Azucar,Pacic,25.9,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes
Azúcar,Bolsa Plástico 900 Gr. Estándar,Zulka,Azucar,Pacic,24.9,03/11/2025,Mega Soriana,Supermercado / Tienda de Autoservicio,Mega Soriana Sucursal Aguascalientes,Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col. Trojes de Alonso. Cp. 20116,Aguascalientes,Aguascalientes


In [ ]:
glimpse(datos)

Rows: 486,182
Columns: 13
$ producto         <chr> "Aceite", "Arroz", "Atún", "Atún", "Azúcar", "Azúcar"…
$ presentacion     <chr> "Botella 850 Ml. Vegetal", "Bolsa 900 Gr. Extra. Gran…
$ marca            <chr> "Precíssimo", "Precíssimo", "Fresh Label", "Fresh Lab…
$ categoria        <chr> "Aceites y Grasas Veg. Comestibles", "Arroz y Cereale…
$ catalogo         <chr> "Pacic", "Pacic", "Pacic", "Pacic", "Pacic", "Pacic",…
$ precio           <dbl> 26.50, 16.90, 17.90, 17.90, 25.90, 24.90, 128.00, 88.…
$ fecha_registro   <chr> "03/11/2025", "03/11/2025", "03/11/2025", "03/11/2025…
$ cadena_comercial <chr> "Mega Soriana", "Mega Soriana", "Mega Soriana", "Mega…
$ giro             <chr> "Supermercado / Tienda de Autoservicio", "Supermercad…
$ nombre_comercial <chr> "Mega Soriana Sucursal Aguascalientes", "Mega Soriana…
$ direccion        <chr> "Aguascalientes Nte. 802. Esq. Blvd. a Zacatecas. Col…
$ estado           <chr> "Aguascalientes", "Aguascalientes", "Aguascalientes",…
$ municipio   

## RAZÓN DE ELECCIÓN

Para mi proyecto de Mercadotecnia Estratégica en Google Colab, voy a analizar la base de datos de Profeco para entender cómo interactúan el pricing, la competencia y la segmentación geográfica en el mercado real. Trabajar con estos 486182 registros me permitirá evaluar la enorme dispersión de precios —como la brecha del 212% en el arroz— entre 63 cadenas comerciales y 30 estados, identificando las estrategias de fijación de precios de cada retail (desde Everyday Low Price hasta High-Low Pricing) y determinando cómo varía la sensibilidad al precio según el giro comercial y la ubicación regional.